app/src/main/java/io/gnosis/safe/ui/settings/app/ScreenshotAuthorizer.kt

```diff
@@ -0,0 +1,23 @@
+package io.gnosis.safe.ui.settings.app
+
+import pm.gnosis.svalinn.common.PreferencesManager
+import pm.gnosis.svalinn.common.utils.edit
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class ScreenshotAuthorizer @Inject constructor(
```

Comment:
don't like naming here. It might be possible that we will manage other privacy-related stuff (maybe something related to tracking). We could rename this class to something like PrivacySettingsHandler and extend it with other settings if the need arises.